src/main/res/values/strings.xml

```diff
@@ -239,6 +239,9 @@ limitations under the License.
     <string name="generic_click_twice_cancel">Click back again to cancel</string>
     <string name="generic_choose_an_option">Choose an option</string>
     <string name="generic_undo">Undo</string>
+    <string name="generic_filter">Filter</string>
+    <string name="generic_from_date">Begin Date</string>
```

Comment:
From?

Reply:
Okay. So I rename generic_from_date to generic_from to make it even more generic :smile: